roles/docker/tasks/cleanup.yml

```diff
@@ -12,3 +12,9 @@
   become: true
   with_items:
     - "{{ docker_api_port }}"
+
+- name: cleanup contiv-compose
```

Comment:
can you please explain, how is it broken?

Assuming `0.0.0-dev (4c0c2dd)` can be a version-variable string and `contiv-compose --version` outputs the right version.